src/utils/vmCodeExecutor.test.js

```diff
@@ -0,0 +1,29 @@
+import executeVmCode from "./vmCodeExecutor";
+
+describe("executeVmCode", () => {
+  it("유저 함수 코드가 정상적으로 실행되어야 한다.", async () => {
+    const fixture = `
+    function userCode(a, b) {
+      return a + b;
+    }
+    userCode(5, 2);
+    `;
```

Comment:
```suggestion
    const fixture = `
        function userCode(a, b) {
            return a + b;
        }
        userCode(5, 2);
    `;
```

사소한 부분이지만 인덴팅 신경써주시면 조금 더 깔끔한 코드가 될 것 같습니다!

Reply:
의견 반영하여 수정하였습니다!